newIDE/app/src/Utils/GDevelopServices/Shop.js

```diff
@@ -9,10 +9,11 @@ const client = axios.create({
   baseURL: GDevelopShopApi.baseUrl,
 });
 
-type StripePrice = {|
+type StripeAndPaypalPrice = {|
   value: number,
-  name: 'default',
+  name: string,
   stripePriceId: string,
```

Comment:
Shouldn't there be a paypalPriceId here?

Reply:
actually no, without going too much detail, it is handled differently on the backend side